src/components/AlertsOverview/AlertsOverview.js

```diff
@@ -0,0 +1,219 @@
+/**
+ * ************************************
+ *
+ * @module  AlertsOverview
+ * @description component that renders basic information about the user's cluster
+ *
+ * ************************************
+ */
+
+import React, { useState, useEffect } from 'react';
+import { AppBar, Box, Toolbar, Container, Typography, Paper } from '@mui/material';
+
+const primaryColor = '#25274D';
+
+// helper function 1: consolidates all alert details from API fetch request into a 1 layer key/value pair of the alert details and groups them by alertname
+function consolidateAlerts(dataArray) {
+  const resultObj = {};
+
+  // if there are any alerts recieved from the fetch request, iterate through each element of the request, and add the details into the resultObj (making the details 1 layer of key/value pairs)
+  if (dataArray.length) {
+    dataArray.forEach(el => {
+      const alertName = el.labels.alertname;
+      const alertDetailsObj = {};
+
+      // if alertname does not exist in resultObj add alertname as the key and set the value to an empty array
+      if (!resultObj[alertName]) {
+        resultObj[alertName] = [];
+      }
+
+      // add alert query "active at", and "state" into the alertDetails Obj
+      const formattedDate = new Date(el.activeAt);
+      alertDetailsObj['Active At'] = `${formattedDate.toLocaleString('en-US', {
+        timeZone: 'America/New_York',
+      })} EST`;
+
+      alertDetailsObj.State = el.state;
+
+      // add each of the alert query annotations key/value pairsinto the alertDetails Obj
+      for (const [key, value] of Object.entries(el.annotations)) {
+        let keyUpdate = key.replace(/[_-]/g, ' ');
+        // ^\w{1} macthes the first letter of the word, | means or, \s+ matches the amt of whitespaces between words
+        // uppercase keys, keys will be leveraged to render to the client
+        keyUpdate = keyUpdate.replace(/(^\w{1})|(\s+\w{1})/g, letter => letter.toUpperCase());
+
+        alertDetailsObj[keyUpdate] = value;
+      }
+
+      //add each of the alert query labels key/value pair into the alertDetails Obj. Except for value and alertname
+      for (const [key, value] of Object.entries(el.labels)) {
+        let keyUpdate = key.replace(/[_-]/g, ' ');
+        keyUpdate = keyUpdate.replace(/(^\w{1})|(\s+\w{1})/g, letter => letter.toUpperCase());
+
+        if (key !== 'alertname' && key !== 'value') {
+          alertDetailsObj[keyUpdate] = value;
+        }
+      }
+
+      // push the object of all other properties into the array corresponding to the alertname
+      resultObj[alertName].push(alertDetailsObj);
+    });
+  }
+
+  return resultObj;
+}
+
+// helper function #2
+function createAlertDetails(el, alertName, i, boxColor) {
+  const objCopy = el;
```

Comment:
This statement doesn't copy the object: both `objCopy` and `el` will point to the same object in memory. Did you mean to deep clone here? If that's the case, I'd recommend `JSON.parse(JSON.stringify(el));`